core-services/release-controller/_releases/release-ocp-4.7.json

```diff
@@ -63,6 +63,17 @@
     "metal-ipi": {
       "prowJob":{"name":"periodic-ci-openshift-release-master-ocp-4.7-e2e-metal-ipi"}
     },
+    "metal-ipi-upgrade":{
+      "upgrade":true,
+      "optional":true,
```

Comment:
The current installation job is gating since 4.7, I'd expect the same for the upgrade as well (so it should be a `blocking` rather than `informing`) at the end

Reply:
@andfasano I know it's supposed to be `blocking` rather than `informing` at the end. I figured that we might want to keep it as informing rather than blocking at first to see that everything is working properly and then change it to blocking if everything works properly.  WDYT? 
Do you want to change it to `blocking` already ? That would just involve removing the `"optional":true`

Reply:
The two-step approach makes sense, ie having it `informing` for a certain amount of time and the moving it as `blocking` later.